src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -15,6 +15,7 @@ internal class LinuxContainerEventGenerator : LinuxEventGenerator, IDisposable
         private readonly Action<string> _writeEvent;
         private readonly BufferedConsoleWriter _consoleWriter;
         private readonly IEnvironment _environment;
+        private readonly int _pid;
```

Comment:
Small nit: you can just inline this: 
```suggestion
        private readonly int _pid = Environment.ProcessId;
```